docs/advanced-features/persistence.md

```diff
@@ -0,0 +1,29 @@
+# Persistence for Static files
```

Comment:
Can we add somewhere that using storage such as S3 is still a recommended way when running a deployment across multiple Kubernetes nodes?

Reply:
Sure, changed the docs. Please check them again!